easybuild/framework/easyconfig/easyconfig.py

```diff
@@ -1285,6 +1285,19 @@ def process_easyconfig(path, build_specs=None, validate=True, parse_only=False,
     return easyconfigs
 
 
+def letter_dir_for(name):
+    """
+    Determine 'letter' directory for specified software name.
+    This usually just the 1st letter of the software name (in lowercase),
+    except for funky software names, e.g. ones starting with a digit.
```

Comment:
well, it should be a catch-all, I don't care how it's called exactly...

I can use `_` instead?